src/Validation/FiValidation.php

```diff
@@ -81,4 +82,133 @@ public static function personId($check)
 
         return $check[strlen($check) - 1] === $list[intval(substr($check, 0, 6) . substr($check, 7, 3)) % 31];
     }
+
+    /**
+     * Checks a country specific business identification number.
+     *
+     * @param string $check The value to check.
+     *
+     * @return bool Success
+     */
+    public static function businessId($check)
+    {
+        $businessId = strtoupper(trim($check));
```

Comment:
What happens with input like `'ÄÖü'` etc? WIll such a string not produce any errors when used with non utf8 methods?

Reply:
Taking a more close look discovered that only numbers can be there in businessId. strtoupper is obsolete